.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
   - "if [ $(phantomjs --version) != '2.1.1' ]; then tar -xvf $PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C $PWD/travis_phantomjs; fi"
   - "phantomjs --version"
 rvm:
-  - 2.2
```

Comment:
This breaks Travis for reasons unclear:
https://travis-ci.org/projecthydra/sufia/builds/126259182#L531